src/main/app/claims/models/response/core/responseAcceptance.ts

```diff
@@ -0,0 +1,17 @@
+import { PaymentIntention } from 'claims/models/response/core/paymentIntention'
+import { CourtDetermination } from 'claims/models/response/core/courtDetermination'
+import { ClaimantResponse } from 'claims/models/response/core/claimantResponse'
+
+export class ResponseAcceptance implements ClaimantResponse {
+  type: string
+  amountPaid: number
+  formaliseOption: string
+  determinationDecisionType: string
```

Comment:
This is renamed to `decisionType` in `claim-store-api`

Reply:
done.